examples/v1beta1/sdk/tune-train-from-func.ipynb

```diff
@@ -466,20 +546,28 @@
     "\n",
     "# Start TFJob Training.\n",
     "tfjob_name = \"train-mnist\"\n",
-    "tfjob_client = TFJobClient()\n",
+    "tfjob_client = TrainingClient(namespace=namespace)\n",
     "\n",
-    "tfjob_client.create_tfjob_from_func(\n",
+    "#create_tfjob_from_func\n",
+    "tfjob_client.create_job(\n",
     "    name=tfjob_name,\n",
-    "    func=train_mnist_model,\n",
+    "    namespace=namespace,\n",
+    "    job_kind=\"TFJob\",\n",
+    "    train_func=train_mnist_model,\n",
     "    parameters=parameters, # Input parameters for the train function.\n",
-    "    num_worker_replicas=5, # How many TFJob Workers will be run.\n",
+    "    num_workers=5, # How many TFJob Workers will be run.\n",
+    "    base_image=\"tensorflow/tensorflow:2.10.0\",  # Use TensorFlow image\n",
```

Comment:
You can skip the base_image, since by default it will used this one: https://github.com/kubeflow/trainer/blob/release-1.9/sdk/python/kubeflow/training/constants/constants.py#L110

Reply:
Did you address this change @yehudit1987 ?